tests/contrib/azure_functions/test_azure_functions_snapshot.py

```diff
@@ -95,6 +95,11 @@ def test_http_get_function_name_no_decorator(azure_functions_client: Client) ->
     assert azure_functions_client.get("/api/httpgetfunctionnamenodecorator", headers=DEFAULT_HEADERS).status_code == 200
 
 
+@pytest.mark.snapshot
+def test_http_get_distributed(azure_functions_client: Client) -> None:
+    assert azure_functions_client.get("/api/httpgetroot", headers=DEFAULT_HEADERS).status_code == 200
```

Comment:
Can there also be a test to show that when distributed tracing is set to `false`, we get disconnected traces?

Reply:
Added a test here: https://github.com/DataDog/dd-trace-py/commit/819ce1b580efa23fbfaa7b80c1d3b5ba9d16ddd6